topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.md

```diff
@@ -0,0 +1,851 @@
+`1/1:255,134,0,255,255,255,255,255,255,255:886:107,773,6,0`.  
+The information provided by the FORMAT field explains the division of the data by colons: `GT:PL:DP:DPR`.
+
+We can break it down like this...
+
+| FORMAT genotype field  | Value                                 |
+|-----------------------:|:--------------------------------------|
+|GT                      |1/1                                    |
+|PL                      |255,134,0,255,255,255,255,255,255,255  |
+|DP                      |886                                    |
+|DPR                     |107,773,6,0                            |
+
+GT = Genotype information, which cannot be used with virus populations!  
+PL = Phred-scaled genotype likelihood (also not useable for us, because we do not have a diploid organism!)
+
+DPR can be broken down even further, since the individual values, which are separated by commas this time, can be assigned to the reference or the alternative nucleotides. In a virus population, four nucleotides (A, T, G and C) can theoretically occur at each position. One nucleotide defines the reference, leaving three alternatives: first (`ALT1`), second (`ALT2`) and third (`ALT3`) alternative. 
+
+
+
+|CHROM   |POS   |DP   |DPR           | ALT1  |ALT2    |ALT3  |  
+|:------:|:----:|:---:|:------------:|:-----:|:------:|:----:|
+|CpGV-M  |246   |886  |107,773,6,0   |773    |6       |0     |
+
+If we now divide the absolute frequencies of `ALT1 = 773`, `ALT2 = 6` and `ALT3 = 0` by `DP = 886`, we get the relative frequencies (`REL.ALT`):
+
+|ALLELE     |REL.ALT     |
+|:---------:|:----------:|
+|ALT1       |0.87246     |
+|ALT2       |0.00677201  |
+|ALT3       |0           |
+
+
+> <hands-on-title> Transfrom VCF to tab-deliminated table </hands-on-title>
+>
+> 1. {% tool [VCFtoTab-delimited:](toolshed.g2.bx.psu.edu/repos/devteam/vcf2tsv/vcf2tsv/1.0.0_rc1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Select VCF dataset to convert"*: `output_file` (output of **bcftools call** {% icon tool %})
+>    - *"Fill empty fields with"*: `NULL`
+>    - Click Run Tool
+>
+>    > <comment-title> An easier-to-read table </comment-title>
+>    > This tool creates a table from the VCF file. The columns are tab-deliminated. Take a look at table and see if it is now easier to read. It can also be imported to R/RStudio/Excel more easily.
```

Comment:
```suggestion
>    > This tool creates a table from the VCF file. The columns are tab-delimited. Take a look at the table and see if it is now easier to read. It can also be imported to R/RStudio/Excel more easily.
```